sql/src/test/java/io/crate/integrationtests/PostgresITest.java

```diff
@@ -633,6 +633,18 @@ public void testSetMultipleSchemasOnSession() throws Exception {
         }
     }
 
+    @Test
+    public void testUseBuiltInFunctionWithSetSchemaOnSession() throws Exception {
```

Comment:
PostgresITest should be kept to postgres protocol specific tests - don't abuse it to set a custom schema.